### DISCUSSION TITLE
Is there a way to save a discussion you're not a part of?

### DISCUSSION THREAD

**Discussion by arjunamrith** [community/community#2854] [Discussions]
I would like to save some answers to refer back to in the future, is it possible to save it/bookmark it in some way?

**Comment by becca** [comment]
That's a great idea! Where would you expect to access saved discussions? 

Notifications allows you to access all of your subscriptions (https://github.com/notifications/subscriptions) but this doesn't support a way to filter on Discussions.

**Reply by arjunamrith** [reply]
I think it would be nice to have it on the [your discussions](https://github.com/discussions) page with a saved discussions/saved answers tab.  It can alternatively be something like the star feature, where you can easily refer back to your stars in the profile dropdown.

**Reply by tomkerkhove** [reply]
I agree, being able to bookmark a discussion to go back to it later on would be great!

**Reply by adam-a-a** [reply]
Right now, you can create a new issue based on a comment to someone's code. 

Similarly, I think adding the analogous option to create a new discussion would be nice to have.

EDIT: just realized this is a bit different from the original poster intended, but that's what I'd be interested in!

**Reply by savornicesei** [reply]
I would personally like a separate section in Issues, not to clutter the main menu too much.

**Comment by jfullstackdev** [comment]
as for now the only thing you can do is to subscribe by clicking the button on the right, this way you'll get notified

![image](https://user-images.githubusercontent.com/47092464/193405212-5915d8ef-e6a8-40e7-8399-912366acabe3.png)

then bookmark the page on top

![image](https://user-images.githubusercontent.com/47092464/193405232-64503f29-ef75-4b2f-aa7f-8666b80885a7.png)

**Comment by candidexmedia** [comment]
+1 for this

**Comment by otabekshoyimov** [comment]
+1

**Comment by WaterSwallow** [comment]
> I would like to save some answers to refer back to in the future, is it possible to save it/bookmark it in some way?

Here are some quick ways to save or bookmark the information:

1. **GitHub Gist**: Go to [GitHub Gist](https://gist.github.com/), create a new gist, paste the information, and save it.

2. **Text File**: Copy the information into a text file using a text editor, and save it on your computer.

3. **Browser Bookmark**: Bookmark this page in your browser for easy access later.

4. **Note-Taking App**: Use apps like Evernote, OneNote, or Notion to create a new note with the information.

5. **Email**: Email the information to yourself with a clear subject line.

**Comment by LiteBrite82** [comment]
You can use the search bar: involves:username and filter from there

**Comment by LilWhiteFoxxx** [comment]
+1
